x-pack/platform/plugins/shared/onechat/server/services/chat/utils/generate_title.ts

```diff
@@ -72,7 +72,7 @@ Conversation:
 
 Now, generate a title for the following conversation.`,
         ],
-        ...conversationToLangchainMessages({ previousRounds, nextInput, ignoreSteps: true }),
+        createUserMessage(nextInput.message),
```

Comment:
I totally changed the way `conversationToLangchainMessages` works so I can no longer re-use it here, but generate_title is only called for the first message at the moment, so I'm not introducing any regression by only create a message for the next input.